gdx/src/com/badlogic/gdx/graphics/g2d/PixmapPacker.java

```diff
@@ -326,66 +322,61 @@ public synchronized void dispose () {
 	 * @return the TextureAtlas */
 	public synchronized TextureAtlas generateTextureAtlas (TextureFilter minFilter, TextureFilter magFilter, boolean useMipMaps) {

```

Comment:
`updateTextureAtlas` is sufficient, but I think `generateTextureAtlas` is a nice convenience function.  It covers what I expect is a pretty common use-case for libgdx games in development: load all the textures into a PixmapPacker, generate the atlas, dispose the packer and then load the skin file.  The programmer doesn't intend to be dynamically adding textures, they're just avoiding having to include the building of a pre-baked texture atlas into the build pipeline.